### PR TITLE
Sprint Final PR

### PR DESCRIPTION
**Pull request by KKonaOG**
This SHOULD fully integrate most of the image acquisition requirements. It is completely untested. A slight remodel of the processing GUI was done to allow the image frame to be used in the cases of both image processing and image acquisition. A new .py was created to move previous team's work into a class that could be used more easily within the GUI, however, its contents should remain mostly the same. Some additional work was completed in an attempt to surplant the need to leverage the GrabDemo for its acquisition dialog. I was not entirely sure what the "Arm" button was going to be used for so I went ahead and wired up the trigger settings popup to that button. We also went ahead and created button for Serial Configuration. As you stated that we should mostly be focusing on Image Acquisition, I went ahead and removed JAISerial implementations and we simply left the buttons as a demonstration on how tab view can be used in the manner it was implemented.

### CODE REVIEW

**Comment by bjones1** [comment]
Would you add `pythonnet` and `pycairo` to `Image-processing-GUI/requirements.txt`? I need those to run the GUI.

...but, the `cairo` import is unused (likewise, `cv2` and `os` are also unused) -- can we just remove these instead?

File: Image-processing-GUI/New_GUI_LEAP.py [hunk 1]
```diff
@@ -2,237 +2,261 @@
...
 
-# Modes: "System" (standard), "Dark", "Light"
+# <p>Modes: "System" (standard), "Dark", "Light"</p>
 customtkinter.set_appearance_mode("Dark")
-# Themes: can be a color ex: "blue", but also a .json file if available in the same folder as the program
+# <p>Themes: can be a color ex: "blue", but also a .json file if available in
+#     the same folder as the program</p>
 customtkinter.set_default_color_theme("Theme.json")
 
+JAISerialHandle = None
```
Comment (bjones1) [inline Image-processing-GUI/New_GUI_LEAP.py]: Aack! Globals! Remove these, unless there's a **really** good reason for them...

File: Image-processing-GUI/New_GUI_LEAP.py [hunk 2]
```diff
@@ -2,237 +2,261 @@
...
+class App(customtkinter.CTk):
+    # <p>Code that affects the GUI's design are put under def __init__(self)
+    #     functions for the widgets are defined under the class, but outside of
+    #     def __init__(self)</p>
     def __init__(self):
         super().__init__()
-
-        # Data Members
-        self.JAIConnection = None
+        self.ImageAcquisitionHandle = ImageAcquisitionManager(self.ImageHandler)
```
Comment (bjones1) [inline Image-processing-GUI/New_GUI_LEAP.py:23]: This is what I'd expected instead of the global.

File: Image-processing-GUI/New_GUI_LEAP.py [hunk 3]
```diff
@@ -2,237 +2,261 @@
...
-    def open_input_dialog_event(self):
-        dialog = customtkinter.CTkInputDialog(
-            text="Type in a number:", title="CTkInputDialog")
-        print("CTkInputDialog:", dialog.get_input())
 
     def sidebar_button_event(self):
         print("sidebar_button click")
 
+    def Freeze(self):
+        if (ImageAcquisitionHandle is not None):
```
Comment (bjones1) [inline Image-processing-GUI/New_GUI_LEAP.py]: Aack! Global! Shouldn't this be `self.ImageAcquisitionHandle` instead? (See also the next line, line 134, etc.)

**Review by bjones1 — CHANGES_REQUESTED** [review]

**Comment by KKonaOG** [comment]
@bjones1 Ready for review

**Comment by bjones1** [comment]
Looks good, thanks!

**Pull request merged**